### PR TITLE
update rpki.texi

### PR DESCRIPTION
**Pull request by colinbs**
The RPKI documentation for the FRR integration needs some work.

### CODE REVIEW

File: doc/rpki.texi [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @c -*-texinfo-*-
-@c This is part of the Quagga Manual.
+@c This is part of the FRRouting Manual.
```
Comment (smlng) [inline doc/rpki.texi]: in `bgpd.texi` this line is `@c This is part of the Frr Manual.` and in `babeld.texi` its `@c This is part of the FRR Manual.`, so inconsistent as it is, I would go for the latter version with `FRR Manual`, or not?

File: doc/rpki.texi [hunk 2]
```diff
@@ -14,15 +14,15 @@ cryptographic data (so called Route Origin Authorization, or short
...
+In combination with a  BGP Prefix Origin Validation scheme a router is able
 to verify received BGP updates without suffering from cryptographic
 complexity.
 
 
 The RPKI/RTR protocol is defined in @cite{RFC6810, The Resource Public Key
 Infrastructure (RPKI) to Router Protocol}, and the validation scheme in
 @cite{RFC6811, BGP Prefix Origin Validation}. The current version of Prefix
-Origin Validation in Quagga implements both RFCs.
+Origin Validation in FRRouting implements both RFCs.
```
Comment (smlng) [inline doc/rpki.texi]: looking through other docs its mostly referred to as plain and short `FRR` not `FRRouting`

File: doc/rpki.texi [hunk 3]
```diff
@@ -74,22 +75,21 @@ state. This allows the creation of local policies, which handle BGP routes
...
-This command enters the RPKI configuration mode. Most commands that start
+This command enables the RPKI configuration mode. Most commands that start
 with @command{rpki} can only be used in this mode.
 
 When it is used in a telnet session, leaving of this mode cause rpki to be initialized.
 
-Please note that executing this command alone does not activate prefix
-validation. You need to add at least one reachable cache server.
+Executing this command alone does not activate prefix
+validation. You need to configure at least one reachable cache server (see next section).
```
Comment (smlng) [inline doc/rpki.texi]: add link to that section instead or additionally.

**Review by smlng — CHANGES_REQUESTED** [review]

**Review by smlng — APPROVED** [review]

**Comment by smlng** [comment]
@colinbs changes look good to me, clear to merge - or are you still editing?

**Comment by colinbs** [comment]
Feel free to merge away : )

**Pull request merged**